packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorModule.m

```diff
@@ -70,25 +71,37 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
   }
   
   @try {
-    
-    // This check was failing, probably because of some race condition
-    // see note at https://developer.apple.com/documentation/foundation/nsfilemanager/1415645-fileexistsatpath?language=objc
-    //    if (![fileManager fileExistsAtPath:path]) {
-    //      reject(@"E_FACE_DETECTION_FAILED", [NSString stringWithFormat:@"The file does not exist. Given path: `%@`.", path], nil);
-    //      return;
-    //    }
-    
     UIImage *image = [[UIImage alloc] initWithContentsOfFile:path];
+    CIImage *ciImage = image.CIImage;
+    if(!ciImage) {
+      ciImage = [CIImage imageWithCGImage:image.CGImage];
+    }
+    ciImage = [ciImage imageByApplyingOrientation:[EXFaceDetectorUtils toCGImageOrientation:image.imageOrientation]];
+    CIContext *context = [CIContext contextWithOptions:@{}];
+    UIImage *temporaryImage = [UIImage imageWithCIImage:ciImage];
+    CGRect tempImageRect = CGRectMake(0, 0, temporaryImage.size.width, temporaryImage.size.height);
+    CGImageRef cgImage = [context createCGImage:ciImage fromRect:tempImageRect];
     
-    NSDictionary *detectionOptions = [EXFaceDetectorUtils defaultFaceDetectorOptions];
-    
-    EXFaceDetector* detector = [[EXFaceDetector alloc] initWithOptions: [EXFaceDetectorUtils mapOptions:detectionOptions]];
-    [detector detectFromImage:image completionListener:^(NSArray<NSDictionary *> * _Nonnull faces, NSError * _Nonnull error) {
+    UIImage *finalImage = [UIImage imageWithCGImage:cgImage];
+    EXFaceDetector* detector = [[EXFaceDetector alloc] initWithOptions: [EXFaceDetectorUtils mapOptions:options]];
+    [detector detectFromImage:finalImage completionListener:^(NSArray<FIRVisionFace *> * _Nonnull faces, NSError * _Nonnull error) {
+      NSMutableArray<NSDictionary*>* reportableFaces = [NSMutableArray new];
+      
+      //      // set correct orientation
+      //      __block UIInterfaceOrientation orientation;
+      
+      if(faces && faces.count > 0) {
```

Comment:
Earlier you declare this variable as `_Nonnull` 🤔

Reply:
Right, forgot that. It needs to be Nullable in fact.